serverless.yml

```diff
@@ -58,13 +58,12 @@ functions:
         - node_modules/**
     events:
       - http:
-          path: /
-          method: ANY
-          authorizer: ${self:custom.authorizer.${self:service}-authorizer}
-      - http:
-          path: /{proxy+}
+          path: '{proxy+}'
           method: ANY
           authorizer: ${self:custom.authorizer.${self:service}-authorizer}
+          cors:
+            origin: '*'
+            cacheControl: 'max-age=600, s-maxage=600, proxy-revalidate'
```

Comment:
I'm not sure if this is required if we have `fastify-cors` as well? ...but I don't think it does any harm either.

Reply:
This adds the OPTIONS method to API Gateway - separate from the other ones so that no authorizer is applied to it

Reply:
Ahh cool, yeah that makes sense.